deploy/helm/kubecf/assets/scripts/jobs/cf-cli-6-linux/setup_internal_security_group.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o xtrace
+
+export PATH="${CF_CLI_PATH}:${PATH}"
+
+until curl --insecure --fail --silent --head "${CF_API}"; do
```

Comment:
```suggestion
until curl --insecure --fail --silent --head "${CF_API}/v2/info"; do
```
We should also add a comment on _why_ poking at the API before UAA is ready is fine.